core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -124,6 +127,12 @@ public void run() {
         routerFactory.addGlobalHandler(LoggerHandler.create(LoggerFormat.DEFAULT));
       }
 
+      routerFactory.addGlobalHandler(
+          CorsHandler.create(buildCorsRegexFromConfig())
+              .allowedHeader("*")
+              .allowedHeader("content-type")
```

Comment:
`*` should cover `content-type` as well right?

Reply:
> Thank you for this! had to hack it up in my branch to make my local setup work :p

Yeah I noticed this when I was playing with the eth2-keymanager-frontend locally that we don't have an option to enable CORS.

Reply:
I would think so. I copied this code from another project so not sure why it was done. Need to finish testing. But agree it doesn't seem make any sense to have both.